### DISCUSSION TITLE
AES256加解密非常慢？有知道怎么解决的吗？

### DISCUSSION THREAD

**Discussion by dte2mdj** [krzyzanowskim/CryptoSwift#951] [General]
do {
  let aes = try AES(key: key, blockMode: CBC(iv: iv), padding: .pkcs7)
  var decryptor = try aes.makeDecryptor()

} catch {
    print(error)
}

**Comment by J-cyz** [comment]
Same Data in different Build Configuration md5 encode have different time, in debug just using 0.5 sec, but in release need 20 sec! confuse -.-?

**Comment by krzyzanowskim** [comment]
what? if anything it supposed to be the other way around

**Comment by J-cyz** [comment]
> 
Looks like block main thread, i even switch async thread, but can't slove.
then i using CommonCrypto to translate, it's working so fast in any Configuration.
